docs/send-data/use-json-configure-sources/json-parameters-hosted-sources.md

```diff
@@ -98,12 +98,29 @@ The Google Workspace Apps Audit Source cannot be created with JSON. This Source
 
 In addition to the common parameters, the following parameters are for an HTTP Source.
 
-| Parameter | Type | Required? | Default | Description | Access |
-|:--|:--|:--|:--|:--|:--|
-| `fields` | JSON Object | No |  | Enable Extended HTTP Metadata Collection by adding `_convertHeadersToFields=true`. | modifiable |
-| `sourceType` | String | Yes |  | HTTP | not modifiable |
-| `messagePerRequest` | Boolean | Yes |  | When set to `true`, only a single message will be sent for each HTTP request. To disable this feature, set to `false`. <br/>You need to specify the common parameter `multilineProcessingEnabled` as false when setting `messagePerRequest` to `true`. | modifiable |
-| `url` | String | No | URL assigned by Sumo for Source | The Source's unique HTTP endpoint web address. | not modifiable |
+###### `fields`
```

Comment:
@kimsauce, I like the older layout better. I find `| JSON Object | Optional | Modifiable |` a little confusing. But if you want to go ahead and change it, go for it.